scripts/minikube.sh

```diff
@@ -67,10 +67,37 @@ function install_minikube() {
     curl -Lo minikube https://storage.googleapis.com/minikube/releases/"${MINIKUBE_VERSION}"/minikube-linux-"${MINIKUBE_ARCH}" && chmod +x minikube && mv minikube /usr/local/bin/
 }
 
+function detect_kubectl() {
+    if type kubectl >/dev/null 2>&1; then
+        command -v kubectl
+        return
+    fi
+    # default if kubectl is not available
+    echo '/usr/local/bin/kubectl'
+}
+
 function install_kubectl() {
+    local latest_version
+    latest_version=$(curl -L https://storage.googleapis.com/kubernetes-release/release/stable.txt 2> /dev/null)
+    if [[ -z "${latest_version}" ]]; then
+        # skip: getting latest version failed, continue with what we have
+        latest_version=${KUBE_VERSION}
+    fi
+    if type "${kubectl}" >/dev/null 2>&1; then
+        local kctl_version version
```

Comment:
instead of `kctl_version`, just use `kubectl_version`

Reply:
sure